scanners/flutter/flutterproject/fileopener.go

```diff
@@ -0,0 +1,25 @@
+package flutterproject
+
+import (
+	"io"
+	"os"
+)
+
+type FileOpener interface {
```

Comment:
I immediately feel like this should be in `go-utils`. Actually, I'm surprised it's not already there.

Reply:
Here is the PR: https://github.com/bitrise-io/go-utils/pull/182

Reply:
The current FileOpener is a bit specific to this use case:
- instead of the original return value of os.Open, which is an os.File, it returns an io.Reader
- instead of the original os.Open, it doesn't produce an error if the file doesn't exist

I wouldn't move this specific version to the go-utils and the more generic solution wouldn't help in this code much, because I still need to implement the same FileOpener, but instead of calling os.Open it would call FileManager.Open.

Because of this, I think we should leave this code as it is, but I think we can still merge the go-utils PR with the generic FileManager.Open.